src/main/resources/fxml/stylesheets/style.css

```diff
@@ -43,4 +43,51 @@
     -fx-background-size: 3em 3em;
     -fx-background-repeat: no-repeat;
     -fx-background-position: center;
+}
+
+.chart-line-mySymbol {
+    -fx-scale-x: 0.7;
+    -fx-scale-y: 0.7;
+}
+
+
+.Total {
+    -fx-stroke: black;
+    -fx-background-color : black;
+}
+
+.chart-legend-item-clicked {
+    -fx-background-color : lightgray;
+    -fx-background-radius: 5px;
+    -fx-padding: 2px;
+}
+
+
+.Pelo_Bianco, .Pelo_Marrone {
```

Comment:
forse è meglio chiamare queste classi in inglese? Dato che nelle risorse e nelle enumerations li abbiamo sempre chiamati in inglese

Reply:
Hai ragione purtroppo però per semplificarmi la vita nel codice ed avere la legenda in Italiano devo per forza fare così, praticamente lo stile è associato a seconda del nome dato alla serie e quindi visualizzato nella legenda per cui se uso le costanti inglesi poi si vedono in inglese

Reply:
Ok, forse si può fare un mapping solo a livello visivo? Proprio nel "testo" della legenda, tipo se ti arriva "Long_Ears" scrivi "Orecchie Lunghe", non so, se ci metti molto lascia stare

Reply:
Il fatto è che la legenda viene riempita in automatico da scalafx in base al nome dato alla serie...

Reply:
Forse è un’idea un po’ azzardata: provare a creare due fxml diversi in italiano e inglese e poi switcharli a seconda della lingua? Forse così verrebbe anche la legenda in inglese.